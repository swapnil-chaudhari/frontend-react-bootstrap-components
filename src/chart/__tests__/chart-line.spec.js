import React from 'react';
import renderShallow from 'render-shallow';
import ChartLine from '../chart-line';

describe('<ChartLine>', () => {
    describe('when it renders', () => {
        let component;

        beforeAll(() => {
            component = renderShallow(
                <ChartLine
                    className="ChartLine-overall"
                    data={ [1, 3, 5, 7, 8, 12, 13, 14] }
                    xScale={ el => 20 * el }
                    yScale={ el => 20 * el }
                />
            ).output;
        });

        test('has proper markup', () => {
            expect(component).toMatchSnapshot();
        });
    });
});
