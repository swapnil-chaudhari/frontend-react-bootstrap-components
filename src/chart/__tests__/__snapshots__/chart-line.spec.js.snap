// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChartLine> when it renders has proper markup 1`] = `
<g>
  <path
    className="ChartLine ChartLine-overall"
    d="M20,20L60,60L100,100L140,140L160,160L240,240L260,260L280,280"
    stroke="#000"
  />
</g>
`;
